go/service/tlf.go

```diff
@@ -60,14 +60,21 @@ func (h *tlfHandler) CompleteAndCanonicalizeTlfName(ctx context.Context, arg key
 	// Append username in case it's not present. We don't need to check if it
 	// exists already since CryptKeys calls below transforms the TLF name into a
 	// canonical one.
-	arg.TlfName = arg.TlfName + "," + string(username)
+	arg.Query.TlfName = arg.Query.TlfName + "," + string(username)
```

Comment:
I feel like whole TLF should be specified by caller.  I know inconvenient for CLI, but it's not for GUI.

Maybe this:  if public, don't do anything.  if private, prepend username.  

The problem is what if `max,chris` and `max#chris` exist and user `chris` types `keybase chat read max`?